dvc/version.py

```diff
@@ -1,6 +1,6 @@
 try:
-    from ._dvc_version import version as __version__  # type: ignore[import]
-    from ._dvc_version import version_tuple  # type: ignore[import]
+    from ._version import version as __version__  # type: ignore[import]
```

Comment:
https://github.com/DiffSK/configobj/issues/203 prevented us from naming this `_version`. But that has been fixed now since [v5.0.7](https://github.com/DiffSK/configobj/pull/228).